contracts/shared/ForwarderXReceiver.sol

```diff
@@ -9,7 +9,10 @@ abstract contract ForwarderXReceiver {
     // The Connext contract on this domain
     IConnext public immutable connext;
 
-    /// Modifier
+    /// EVENTS
+    event ForwardedFunctionCallFailed();
```

Comment:
what failure modes exist for `xReceive`? they should be documented, as well as what happens to the funds associated with the transfer (if any)

Reply:
if you want to use this as a template, these things must be spelled out clearly

Reply:
Docs coming soon!